inst/ms/aggreCAT.qmd

```diff
@@ -1581,105 +1668,66 @@ rapidly and easily analysing the results of IDEA protocol and other
 structured elicitation procedures where mathematical aggregation of
```

Comment:
L1649: Well we're finished with phase two now...

Reply:
Not sure what you mean sorry!

Reply:
Yes, I agree, good pick up!

Reply:
Also, what is the total number of direct replications?

Reply:
> Score for over 3000 SBS research claims for phase one and **X** SBS
claims for phase two of the DARPA SCORE project.

- [ ] Total claims for whole project please somebody?

Reply:
- [ ] rm ref to phases
- Phase 2: 900 single trace + 100 bushel papers (100 papers were from original 3000 single-trace)
- phase 1: 3000 single trace + 100 covid.

Total (if double-count) bushels/100 from single-trace: 4100, otherwise 4000.
- "In excess of 4000 claims across the whole project"